IBPSA/Fluid/Storage/Examples/StratifiedUnloadAtMinimumTemperature.mo

```diff
@@ -144,18 +145,20 @@ equation
           {48,112}},color={255,0,255}));
   connect(yTop.y, valTop.y) annotation (Line(points={{101,120},{122,120},{122,-8}},
                                color={0,0,127}));
-  connect(yMid.y, valMed.y) annotation (Line(points={{101,80},{142,80},{142,-48}},
+  connect(yMid.y,valMid. y) annotation (Line(points={{101,80},{142,80},{142,-48}},
                               color={0,0,127}));
   connect(not1.u, onOffBot.y) annotation (Line(points={{-22,60},{-26,60},{-26,40},
           {-29,40}},       color={255,0,255}));
   connect(not1.y, and2.u2) annotation (Line(points={{1,60},{4,60},{4,72},{8,72}},
         color={255,0,255}));
-  connect(hea.port, tan.heaPorVol[5]) annotation (Line(points={{-130,-124},{-110,
-          -124},{-110,-120}}, color={191,0,0}));
-  connect(TTop.port, tan.heaPorVol[1]) annotation (Line(points={{-160,-90},{-110,
-          -90},{-110,-120}}, color={191,0,0}));
+  connect(hea.port, tan.heaPorVol[5]) annotation (Line(points={{-130,-124},{
+          -110,-124},{-110,-119.76}},
+                              color={191,0,0}));
+  connect(TTop.port, tan.heaPorVol[1]) annotation (Line(points={{-160,-90},{
+          -110,-90},{-110,-120.24}},
+                             color={191,0,0}));
```

Comment:
As explained in the review comment. If not wanted, revert. Else, ignore.

Reply:
These are reverted in 6ee8e0d.